app/views/components/fields/field_accessible_autocomplete_with_header.scala.html

```diff
@@ -47,7 +47,7 @@
 
     @if(isPageHeading) {<h1 class="govuk-label-wrapper">}
     <label id="@(fieldNameWithoutBrackets)-label" for="@fieldNameWithoutBrackets" class="govuk-label@if(labelClass.nonEmpty){ @labelClass}" >
-        @if(label.nonEmpty) { @messages(label) }
+        @messages(label)
```

Comment:
This was a tmp. fix added to prevent tests failing when trying to de-code empty message key